exercises/C-comparison-operators/exercise.js

```diff
@@ -7,14 +7,14 @@
 
 let studentCount = 16;
 let mentorCount = 9;
-let moreStudentsThanMentors; // finish this statement
+let moreStudentsThanMentors = true; // finish this statement
```

Comment:
```suggestion
let moreStudentsThanMentors = studentCount > mentorCount; // finish this statement
```

Reply:
Hi Micheale, I didn't think of your solution at all. Nice! My suggestion is because I think if we wanted moreStudentsThanMentors to change each week depending on the number of students and mentors present, it would change automatically as studentCount and mentorCount were updated.